framework/console/controllers/MigrateController.php

```diff
@@ -301,11 +301,9 @@ protected function truncateDatabase()
 
         // First drop all foreign keys,
         foreach ($schemas as $schema) {
-            if ($schema->foreignKeys) {
```

Comment:
Are you sure foreignKeys is always iterable?

Reply:
I've based this on docs but you are right, with Yii you never know :P Anyway, I've just run search and look at all the mentions for that and I'm 99% sure.